README.md

```diff
@@ -0,0 +1,31 @@
+# Octavia Provider Driver for F5 BigIP LTM devices
+
+This is the [Octavia](https://github.com/sapcc/octavia) provider driver for F5 BigIP LTM appliances.
```

Comment:
You can just say BigIP appliances, LTM is just a software/license module providing loadbalancing for it. Although this driver is specifically for the LTM modul.